src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/ClientWebSocketOptions.cs

```diff
@@ -46,6 +48,8 @@ public Version HttpVersion
             }
         }
 
+        /// <summary>Gets or sets the policy that determines how <see cref="P:System.Net.WebSockets.ClientWebSocketOptions.HttpVersion" /> is interpreted and how the final HTTP version is negotiated with the server.</summary>
+        /// <value>The HttpVersionPolicy used when the HTTP connection is established.</value>
```

Comment:
Maybe natural language sounds better here?:
```suggestion
        /// <value>The version policy used when the HTTP connection is established.</value>
```